### ISSUE TITLE
jQuery version dependency?

### ISSUE THREAD

**Issue by CaspianRoach** [digitalBush/jquery.maskedinput#313]
You should add information about which jQuery version the plugin requires on the main site. Working with legacy CMS you sometimes have to use the older jQuery versions so it would be nice to know if it's compatible.

**Comment by digitalBush** [comment]
Good point. I have no idea what version is required. I'll need to do some digging and see what versions the syntax we are using got introduced.

**Comment by djpowers** [comment]
FWIW, I'm working on a project using jQuery 1.7.1. I was unable to get the `1.4.x` versions to work with it, but `1.3.1` has no issues.
